Maestro/subscriptions.json

```diff
@@ -509,7 +509,10 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/arcade/blob/master/**/*",
-        "https://github.com/dotnet/machinelearning/blob/master/**/*"
+        "https://github.com/dotnet/machinelearning/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.0/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.1/**/*",
```

Comment:
They also need the `arm64` branch mirrored. I'll add more context to the core-eng issue.

Reply:
Added arm64.